src/components/App.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import {withRouter} from 'react-router';
+import toastr from 'toastr';
+import Constants from '../constants';
+class App extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            isOfflineMsg:false
```

Comment:
what is the purpose of isOfflineMsg? It's not used anywhere.
Seems like a simple component, try converting it into functional component

Reply:
removed state variable isOfflineMsg dependency